e2e/cypress/tests/integration/collapsed_reply_threads/global_threads_spec.js

```diff
@@ -0,0 +1,330 @@
+                CollapsedThreads: 'default_off',
+            },
+        });
+
+        // # Create new channel and other user, and add other user to channel
+        cy.apiInitSetup({loginAfter: true, promoteNewUserAsAdmin: true}).then(({team, channel, user}) => {
+            testTeam = team;
+            user1 = user;
+            testChannel = channel;
+
+            // # Visit the channel
+            cy.visit(`/${testTeam.name}/channels/${testChannel.name}`);
+
+            // # close the trial modal
+            cy.get('#startTrialModal', {timeout: TIMEOUTS.HALF_MIN}).find('.modal-header button.close').first().click();
+
+            cy.apiSaveCRTPreference(user1.id, 'on');
+            cy.apiCreateUser({prefix: 'user2'}).then(({user: newUser}) => {
+                user2 = newUser;
+
+                cy.apiAddUserToTeam(testTeam.id, user2.id).then(() => {
+                    cy.apiAddUserToChannel(testChannel.id, user2.id);
+                });
+            });
+
+            cy.apiCreateUser({prefix: 'user3'}).then(({user: newUser}) => {
+                user3 = newUser;
+
+                cy.apiAddUserToTeam(testTeam.id, user3.id).then(() => {
+                    cy.apiAddUserToChannel(testChannel.id, user3.id);
+                });
+            });
+        });
+    });
+
+    beforeEach(() => {
+        // # Visit the channel
+        cy.visit(`/${testTeam.name}/channels/${testChannel.name}`);
+
+        // # Post a message as other user
+        cy.postMessageAs({sender: user1, message: messages.ROOT, channelId: testChannel.id}).then((post) => {
+            rootPost = post;
+        });
+    });
+
+    it('MM-T4379: Display: Click to open threads)', () => {
+        cy.uiWaitUntilMessagePostedIncludes(rootPost.data.message);
+
+        // # Get the root post
+        cy.getLastPost().click();
+
+        // * Verify the post is opened in RHS
+        cy.get(`#rhsPost_${rootPost.id}`).should('be.visible');
+
+        // # close RHS
+        cy.uiCloseRHS();
+
+        // # open settings modal and disable opening the thread in RHS when being clicked
+        cy.uiOpenSettingsModal('Display');
+
+        // # open section "click to open threads"
+        cy.get('#click_to_replyTitle').click();
+
+        // # click radio button with option B ('off')
+        cy.get('#click_to_replyFormatB').click();
+
+        // # save settings
+        cy.get('#saveSetting').click();
+
+        // # close settings modal
+        cy.uiClose();
+
+        // # (re-) Visit the channel
+        cy.visit(`/${testTeam.name}/channels/${testChannel.name}`);
+
+        // # Get the root post
+        cy.getLastPost().click();
+
+        // * Verify the post is opened in RHS
+        cy.get(`rhsPost_${rootPost.id}`).should('not.exist');
+
+        // # Cleanup for next test
+        cy.apiDeletePost(rootPost.id);
+    });
+
+    it('MM-T4445: CRT - Delete root post (current behavior is incorrect, see comments)', () => {
```

Comment:
I'd suggest to remove `(current behavior is incorrect, see comments)` and add in the succeeding comments instead. Same with others.